.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
+  or if the issue is actually from `libmongoc` or `libbson`. If so, you should
+  file the issue with the representative projects.
+-->
+
```

Comment:
I'd suggest we condense the following items into something shorter, like

1. What version of Swift are you using? (Run `swift --version`)
2. What operating system are you using? (Run `uname -a`)
3. What versions of the driver and its dependencies are you using? (Run `swift package show-dependencies`)
4. What version of MongoDB are you using? (Check with the MongoDB shell using `db.version()`) 
5. What is your MongoDB topology (standalone, replica set, sharded cluster, serverless)?